scripts/nitro-testnode.sh

```diff
@@ -41,10 +48,10 @@ then
   cd "$MYDIR" || exit
   cd ..
 
-  git clone --recurse-submodules https://github.com/OffchainLabs/nitro-testnode.git
+  git clone -b release --recurse-submodules https://github.com/OffchainLabs/nitro-testnode.git
   cd ./nitro-testnode || exit
-  # `release` branch.
-  git checkout 8cb6b84e31909157d431e7e4af9fb83799443e00 || exit
+  git pull origin release --recurse-submodules
+  git checkout 43861b001713db3526b74e905e383d41e9272760 || exit
```

Comment:
Not important.... But do we need to clone the release branch if we are doing a checkout on a specific commit? Why don't we just checkout and run from release branch? Is it because we are afraid of breaking changes?(I still think that it is probably better find out right away)

Reply:
After our talk today do you want to just run the latest from the release branch or prefer to keep the specific commit? Regardless this looks ready to go!

Reply:
I've just found one downside of this approach applicable for benchmarks. You can check out how gas fees have changed after I've updated the nitro test node with a bug fix. Improvement is significant. And since I've just lifted the nitro-test-node's version it is obvious what is the reason.
But in future when we will see some random uplift/downgrade of benchmarks, we can whether blame our library changes or hidden changes at nitro test node.
I think it is better to keep the commit hash for now. And in future record in benchmark's report metadata about nitro-test-node revision. And use just a release branch. That will help us to analyze why performance improved/degraded
Wdt?